[RHACS] ROX-19745: Updating build time network policy tools documentation (#64889)
openshift/openshift-docs: modules/exploring-the-options-for-the-roxctl-netpol-connectivity-diff-command.adoc

// Module included in the following assemblies:
//
// * operating/build-time-network-policy-tools.adoc

:_content-type: REFERENCE
[id="exploring-the-options-for-the-roxctl-netpol-connectivity-diff-command_{context}"]
= Exploring the options for the roxctl netpol connectivity diff command

The `roxctl netpol connectivity diff` command supports the following options:

[%autowidth,options="header"]
|===

|Option |Description

|`--dir1 string`
|First directory path of the input resources. This is a mandatory option.

|`--dir2 string`
|Second directory path of the input resources to be compared with the first directory path. This is a mandatory option.

|`--fail`
|Fail on the first encountered error. The default value is `false`.

|`-h`, `--help`
|View the help text for the `roxctl netpol connectivity diff` command.

|`-f`, `--output-file string`
|Save the connections difference output into a specific file.

|`-o`, `--output-format string`

> |Configure the connections differences in a specific format. The supported formats are `txt`, `md`, and `csv`. The default value is `txt`.

Reviewer: ```suggestion
|Configure the output format. The supported formats are `txt`, `md`, and `csv`. The default value is `txt`.
```

Author: obsolete module
